src/org/omegat/util/Log.java

```diff
@@ -297,31 +294,148 @@ public static void logErrorRB(String key, Object... parameters) {
      *            StaticUtils.format.
      */
     public static void logErrorRB(Throwable ex, String key, Object... parameters) {
-        if (LOGGER.isLoggable(Level.SEVERE)) {
-            LogRecord rec = new LogRecord(Level.SEVERE, key);
-            rec.setResourceBundle(OStrings.getResourceBundle());
-            rec.setParameters(parameters);
-            rec.setLoggerName(LOGGER.getName());
-            rec.setThrown(ex);
-            LOGGER.log(rec);
-        }
+        LOGGER.atError().log(getMessage(key, parameters), ex);
     }
 
     /**
-     * Writes debug message to log (without localization)
+     * Writes a debug message to log (without localization).
      *
      * @param message
      *            message text
      * @param parameters
      *            Parameters for the error message. These are inserted by using
      *            StaticUtils.format.
      */
-    public static void logDebug(Logger logger, String message, Object... parameters) {
+    @Deprecated
+    public static void logDebug(java.util.logging.Logger logger, String message, Object... parameters) {
         if (logger.isLoggable(Level.FINE)) {
             LogRecord rec = new LogRecord(Level.FINE, message);
             rec.setParameters(parameters);
             rec.setLoggerName(logger.getName());
             logger.log(rec);
         }
     }
+
+    public static String getMessage(String key) {
+        return OStrings.getString(key) + " (" + key + ")";
+    }
+
+    public static String getMessage(String key, Object arg) {
+        String message = OStrings.getString(key);
+        return message.replace("{0}", String.valueOf(arg)) + " (" + key + ")";
+    }
+
+    public static String getMessage(String key, Object... args) {
+        String message = OStrings.getString(key);
+        return StringUtil.format(message, args) + " (" + key + ")";
+    }
```

Comment:
We have `OStrings.getString` that mostly duplicated with `Log.getMessage`

`Ostrings.getString` is
```java
    /**
     * Returns a localized string for a key, formatted with the given arguments. Formatting is done by
     * {@link StringUtil#format(String, Object...)}.
     */
    public static String getString(String key, Object... formatArgs) {
        return StringUtil.format(getString(key), formatArgs);
    }

```